### PR TITLE
DR-2715: Fixing vercel logging error

### PR DESCRIPTION
**Pull request by 7emansell**
## Ticket:

Fixes JIRA ticket [DR-2715](https://jira.nypl.org/browse/DR-2715) (particularly its Vercel preview).

## This PR does the following:
- Creates an API route (serverless) that allows Vercel to create the log directory at runtime
- Log directory does not persist
- Needs refactoring since the endpoint is hardcoded

### Checklist:

- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: logger.js [hunk 1]
```diff
@@ -51,6 +51,16 @@ const formatter = printf((info) => {
   return JSON.stringify(logObject);
 });
 
+// API workaround for creating log directory in Vercel
+import axios from "axios";
+
+const logDirCreationEndpoint =
+  "https://digital-collections-git-feature-dr-2715logging-nypl.vercel.app/api/createLogDir";
```
Comment (EdwinGuzman) [inline logger.js:58]: This should work if it's `/api/createLogDir`.

**Comment by EdwinGuzman** [comment]
The previous PR/feature worked on AWS and Cloudwatch but not on Vercel? Did Vercel throw errors somewhere or how did this issue come up?

**Comment by 7emansell** [comment]
@EdwinGuzman exactly– logger worked locally as well as on qa-dcfrontend, and logged to Cloudwatch as expected. On Vercel, the build succeeds, so nothing was flagged when I pushed it up, but when you actually view the preview it fails with a 500 error (I guess I hadn't thought to check the preview because I wasn't making any visual changes). There's not loads of documentation but seemingly Vercel needs to use a serverless function to read or write to the file system. This is a possible workaround Ali and I discovered on Friday

File: logger.js (hunk 1 above)
Comment (7emansell) [reply logger.js:58]: As in `const logDirCreationEndpoint = '/api/createLogDir'` ?

**Pull request closed without merging**